### PR TITLE
Fix line break when selecting HA

### PR DESCRIPTION
**Pull request by fiunchinho**
I was getting a weird line break, so I tested the template in the playground and I belive it's not correct. You can see it in action here https://play.golang.org/p/SNTCNgB4mHp

## Checklist

- [X] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Review by calvix — APPROVED** [review]
lgtm iwfy

File: files/manifests/k8s-api-server.yaml [hunk 1]
```diff
@@ -26,9 +26,9 @@ spec:
     {{ range .Kubernetes.Apiserver.CommandExtraArgs -}}
     - {{ . }}
     {{ end -}}
-    {{- if .Etcd.HighAvailability }}
+    {{- if .Etcd.HighAvailability -}}
```
Comment (tfussell) [inline files/manifests/k8s-api-server.yaml:29]: This doesn't collapse too much whitespace (e.g. putting multiple flags on one line) when used with or without `.Kubernetes.Apiserver.CommandExtraArgs`?

Comment (fiunchinho) [reply files/manifests/k8s-api-server.yaml:29]: I think it's fine. Take a look at the updated example https://play.golang.org/p/DxKxmnrURtG

Comment (tfussell) [reply files/manifests/k8s-api-server.yaml:29]: Cool, thanks.

**Review by tfussell — APPROVED** [review]

**Pull request merged**